modules/nested-accordion/assets/js/frontend/handlers/nested-accordion.js

```diff
@@ -62,23 +62,41 @@ export default class NestedAccordion extends Base {
 	}
 
 	linkContainer( event ) {
-		const { container } = event.detail,
+		const { container, targetIndex } = event.detail,
 			view = container.view.$el,
 			id = container.model.get( 'id' ),
 			currentId = this.$element.data( 'id' );
 
 		if ( id === currentId ) {
-			const containers = view.find( this.getSettings( 'selectors.accordionContentContainers' ) ),
-				accordionItems = view.find( this.getSettings( 'selectors.accordionItems' ) ),
-				lastContentContainer = containers[ containers.length - 1 ],
-				lastAccordionItem = accordionItems[ accordionItems.length - 1 ];
+			const [ accordionItem, contentContainer ] = 'undefined' === typeof targetIndex ? this.insert( view ) : this.move( view, targetIndex );
```

Comment:
We probably will have to discuss how this works with our change for the duplication.